src/group_theory/group_action/units.lean

```diff
@@ -32,6 +32,10 @@ instance [monoid M] [has_scalar M α] : has_scalar Mˣ α :=
 lemma smul_def [monoid M] [has_scalar M α] (m : Mˣ) (a : α) :
   m • a = (m : M) • a := rfl
 
+@[simp] lemma smul_is_unit [monoid M] [has_scalar M α] {m : M} (hm : is_unit m) (a : α) :
```

Comment:
Should this be called `is_unit.unit_smul` rather than `units.smul_is_unit`? The goal doesn't contain `is_unit`, so the current name seems bad.